pkg/build/node/import.go

```diff
@@ -50,7 +50,7 @@ func (c *containerdImporter) Prepare() error {
 
 func (c *containerdImporter) LoadCommand() exec.Cmd {
 	return c.containerCmder.Command(
-		"ctr", "--namespace=k8s.io", "images", "import", "--no-unpack", "-",
+		"ctr", "--namespace=k8s.io", "images", "import", "--all-platforms", "--no-unpack", "-",
```

Comment:
TODO: ideally we do not need this in the future. we have fixed at least one image

Reply:
Yes, this just bypasses the PLATFORM error. If all necessary images have correct PLATFORM in their image config, we can drop this.

Reply:
> TODO: ideally we do not need this in the future. we have fixed at least one image
Yes, this just bypasses the PLATFORM error. If all necessary images have correct PLATFORM in their image config, we can drop this.

Reply:
> TODO: ideally we do not need this in the future. we have fixed at least one image

Yes, this just bypasses the PLATFORM error. If all necessary images have correct PLATFORM in their image config, we can drop this.